abi-versions/v0.2.1/README.md

```diff
@@ -2081,6 +2081,13 @@ changes to unrelated connections/requests.
 - `MONOTONIC` = `1`
 
 
+# Changes from [v0.2.0]
```

Comment:
I'm not sure if this is best placed at the end of the beginning of the file. Happy to change it if someone has any preference.

Reply:
Another option would be to have the changelog in spec/abi-versions/README.md

Reply:
Good point. I think [CHANGELOG.md](https://keepachangelog.com/en/1.1.0/) (which we already use in Rust SDK) in the main directory might be even better.

Reply:
Agreed!